src/app/core/components/header/user-panel/user-panel.component.html

```diff
@@ -1,10 +1,10 @@
-<div class="user-actions-container">
-  <div class="user-login-label">
-    <fa-icon class="user-action-icon" [icon]="iconUser"></fa-icon>
-    <span>User login</span>
+<div class="user-panel">
```

Comment:
something like if userIsLoggedIn -> show name and logout -> else -> shon login and no name...

Reply:
Okay